src/vector/platform/ElectronPlatform.tsx

```diff
@@ -575,9 +575,9 @@ export default class ElectronPlatform extends VectorBasePlatform {
     onKeyDown(ev: KeyboardEvent): boolean {
         let handled = false;
 
-        switch (ev.key) {
-            case Key.SQUARE_BRACKET_LEFT:
-            case Key.SQUARE_BRACKET_RIGHT:
+        switch (ev.code) {
+            case "BracketLeft":
+            case "BracketRight":
```

Comment:
changing these definitely feels wrong as the KeyboardShortcuts dialog will be out of sync and very misleading.